Lib/test/test_socket.py

```diff
@@ -4276,9 +4276,13 @@ def testSetBlocking_overflow(self):
         if _testcapi.UINT_MAX >= _testcapi.ULONG_MAX:
             self.skipTest('needs UINT_MAX < ULONG_MAX')
         self.serv.setblocking(False)
+        if fcntl:
+            self.assertFalse(_is_fd_in_blocking_mode(self.serv))
         self.assertEqual(self.serv.gettimeout(), 0.0)
         self.serv.setblocking(_testcapi.UINT_MAX + 1)
```

Comment:
The NEWS entry mention 2**32, but you use 2**32+1. You may repeat this test on UINT_MAX and UINT_MAX+1. You may use a simple loop or a subfunction and "with subTest()".

Reply:
`UINT_MAX + 1 == 2**32`, so I don't :)

OK, I'll make an extra test.

Reply:
Oh. In that case, no new test is needed.

Reply:
I was thinking about a test checking that `UINT_MAX+1` is treated as `True`, but if the changes need to be minimal, OK.

Reply:
BTW, Travis failed with the following error in the very end:
```
xvfb-run:
   error:
   Xvfb
   failed
   to
   start
```